createlinks

```diff
@@ -38,6 +38,21 @@ event_templates('nethserver-cockpit-update', qw(
     /etc/nethserver/ldappasswd.conf
     /etc/sudoers.d/30_nethserver_cockpit_roles
     /etc/sudoers.d/55_nsapi_perms
+    /etc/nethserver/cockpit.allow
+    /etc/pam.d/cockpit
+));
+
+# expand cockpit PAM following `ShellOverrideStatus`
+event_templates('nethserver-sssd-save', qw(
+    /etc/pam.d/cockpit
+));
+
+# delegate cockpit access via PAM
+event_templates('group-modify', qw(
+    /etc/nethserver/cockpit.allow
```

Comment:
I can't figure out a use case where this is needed, both for `group-modify` and `group-create`. Instead the `group-delete` could be interesting. But it would work only with a local account provider.

Reply:
you are right we miss the group delete, we have to maintain a list following the permission stored under ${cockpit.socket}{delegation}, so with group-{create,delete,modify} it entends to add or remove permission to login to cockpit

For a remote account provider, it is out of the scope of this issue, but our group panel should allow to add permission locally even if the group is remote

Reply:
an alternative is to use `delegation-save`. I will commit to use it

Reply:
:thinking: 

1. group-create: during this event no delegation occurs, for sure. 

2. group-modify: here too

3. group-delete: it _could_ update the `delegation` prop but it only works with a local accounts provider. Furthermore, if a group does not exist I think there is no problem because no user can match PAM/sshd rules with it.

Instead, we'd bind the `delegation-save` event.

Reply:
done we expand the list of group with `delegation-save`